### DISCUSSION TITLE
link-decoration default to underline

### DISCUSSION THREAD

**Discussion by 0x11DFE** [twbs/bootstrap#32276] [Q&A] [answered]
Hello, I was trying to switch from bootstrap 4.5.2 to 5.0.0 and all my links had underline was wondering what's up and i found that the default link-decoration was changed to underline. Is is on purpose or i should open an Issue?

**Comment by patrickhlauke** [comment]
it's on purpose (though i notice that it was not deemed breaking enough to make it into the migration document) https://github.com/twbs/bootstrap/pull/30389

**Comment by XhmikosR** [comment]
@twbs/css-review care to add it to the migration notes?

**Comment by patrickhlauke** [comment, marked as answer]
done (also noticed the typography page still had the misleading bit about underlines only on hover)

https://github.com/twbs/bootstrap/pull/32281/
